lib/artisanal/form/dsl.rb

```diff
@@ -1,27 +1,24 @@
 require 'active_support/core_ext/hash/deep_merge'
 
 module Artisanal::Form
-  require_relative 'coordinator'
+  require_relative 'attribute'
```

Comment:
Most of the changes in this class are either re-ordering the methods alphabetically or moving them into the new `Artisanal::Form::Form` class.

The only new methods are `artisanal_form` on the class and instance and the `register_context` method (which is delegated to the form instance).